api/models/tea.model.ts

```diff
@@ -1,11 +1,35 @@
 export interface InputVarMod {
```

Comment:
Divide input into GenericPowerOnly input and GasificationPower input. You could divide InputVarMod and make all GenericPowerOnly input as GenericPowerOnlyInputMod and all GasificationPower input as GasificationPowerInputMod.